src/bitcoin-cli.cpp

```diff
@@ -29,29 +29,26 @@ static const int DEFAULT_HTTP_CLIENT_TIMEOUT=900;
 static const bool DEFAULT_NAMED=false;
 static const int CONTINUE_EXECUTION=-1;
 
-static std::string HelpMessageCli()
+void SetupArgs()
```

Comment:
Nit: More suitable name could be `static void SetupCliArgs`. Including the static after #13163